HW04/anastasia_evsihneyeva/lib/notification.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Notification
+  attr_accessor :logs
+
+  def initialize(text = nil)
+    @logs =  []
+    @logs << text if text
+  end
+
+  def mark_as_read!
+    @logs.clear
+  end
```

Comment:
This is not the responsibility of this class (transfer this to the mentor and student class)